src/main/native/cpp/view/AnalyzerPlot.cpp

```diff
@@ -370,6 +389,12 @@ void AnalyzerPlot::DisplayTimeDomainPlots(ImVec2 plotSize) {
       m_fitNextPlot[6] = false;
     }
   }
+  return true;
+}
+
+bool AnalyzerPlot::LoadPlots() {
+  // See if the plots are loaded
+  return DisplayVoltageDomainPlots() && DisplayTimeDomainPlots();
```

Comment:
Instead of calling these both at once, can we combine it all into one method? We don't need the ImVec2 parameter either.

Reply:
Actually, let's keep these separate so there isn't a huge method. But we can remove the parameter.

Reply:
Where are we removing the parameter from?

Reply:
The ImVec2 parameter to DisplayVoltageDomainPlots() and DisplayTimeDomainPlots().

Reply:
How would we distinguish between the popup version and the regular version of the graphs?

Reply:
There is only one version now right?

Reply:
There are two: this invocation with a default plot size and DisplayCombinedPlots()'s invocation with a non-default plot size.

Reply:
Should we just do one method? Or do both work in this case since they do serve different purposes?

Reply:
Ah, I wasn't aware that we still wanted both methods. 👍